<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-12T21:47:24 CST" name="Default suite" finished-at="2021-03-12T21:47:42 CST" duration-ms="17346">
    <groups>
    </groups>
    <test started-at="2021-03-12T21:47:24 CST" name="Default test" finished-at="2021-03-12T21:47:42 CST" duration-ms="17346">
      <class name="tests.TestsLogin">
        <test-method is-config="true" signature="setUpDriver()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:24 CST" name="setUpDriver" finished-at="2021-03-12T21:47:28 CST" duration-ms="3454" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpDriver -->
        <test-method signature="testLogin()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:28 CST" name="testLogin" finished-at="2021-03-12T21:47:29 CST" duration-ms="1205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method is-config="true" signature="finishTests()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:29 CST" name="finishTests" finished-at="2021-03-12T21:47:30 CST" duration-ms="486" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finishTests -->
        <test-method is-config="true" signature="setUpDriver()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:30 CST" name="setUpDriver" finished-at="2021-03-12T21:47:32 CST" duration-ms="2974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpDriver -->
        <test-method signature="testIncorrectPassword()[pri:1, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:32 CST" name="testIncorrectPassword" finished-at="2021-03-12T21:47:33 CST" duration-ms="538" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null
at pages.AssertHomePage.isDisplayed(AssertHomePage.java:16)
at tests.TestsLogin.testIncorrectPassword(TestsLogin.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIncorrectPassword -->
        <test-method is-config="true" signature="finishTests()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:33 CST" name="finishTests" finished-at="2021-03-12T21:47:34 CST" duration-ms="522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finishTests -->
        <test-method is-config="true" signature="setUpDriver()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:34 CST" name="setUpDriver" finished-at="2021-03-12T21:47:37 CST" duration-ms="3072" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpDriver -->
        <test-method signature="testIncorrectUsername()[pri:2, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:37 CST" name="testIncorrectUsername" finished-at="2021-03-12T21:47:37 CST" duration-ms="528" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null
at pages.AssertHomePage.isDisplayed(AssertHomePage.java:16)
at tests.TestsLogin.testIncorrectUsername(TestsLogin.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIncorrectUsername -->
        <test-method is-config="true" signature="finishTests()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:37 CST" name="finishTests" finished-at="2021-03-12T21:47:38 CST" duration-ms="508" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finishTests -->
        <test-method is-config="true" signature="setUpDriver()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:38 CST" name="setUpDriver" finished-at="2021-03-12T21:47:41 CST" duration-ms="2979" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpDriver -->
        <test-method signature="testIncorrectCredentials()[pri:3, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:41 CST" name="testIncorrectCredentials" finished-at="2021-03-12T21:47:41 CST" duration-ms="538" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.isDisplayed()" because "this.navUserName" is null
at pages.AssertHomePage.isDisplayed(AssertHomePage.java:16)
at tests.TestsLogin.testIncorrectCredentials(TestsLogin.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIncorrectCredentials -->
        <test-method is-config="true" signature="finishTests()[pri:0, instance:tests.TestsLogin@e7edb54]" started-at="2021-03-12T21:47:41 CST" name="finishTests" finished-at="2021-03-12T21:47:42 CST" duration-ms="514" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finishTests -->
      </class> <!-- tests.TestsLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
